src/pages/HealthContent.tsx

```diff
@@ -1,10 +1,133 @@
 import React from "react";
+// @ts-ignore
```

Comment:
Why `ts-ignore` here? Does it throw errors? If so, there's better ways to fix/handle it. Importing images is a common thing that TS definitely has a way to handle.